cmake/Dependencies.cmake

```diff
@@ -1380,8 +1380,6 @@ if(USE_UCC)
   if(NOT CMAKE_SYSTEM_NAME STREQUAL "Linux")
     message(WARNING "UCC is currently only supported under Linux.")
     caffe2_update_option(USE_UCC OFF)
-  else()
-    include(${CMAKE_CURRENT_LIST_DIR}/External/ucc.cmake)
```

Comment:
I guess we will need to add ucc.cmake back when we need to support USE_SYSTEM_UCC=0.

Reply:
This is a good point. I added it back.